assignments/objects.js

```diff
@@ -18,27 +18,73 @@ let example = {
 }
 
 // Write your intern objects here:
+let mitzi = {
+  id: 1,
+  name: "Mitzi",
+  email: "mmelloy0@psu.edu",
+  gender: "F"
+}
+
+let kennan = {
+  id: 2,
+  name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M",
+  speak: function (){
+    return "Hello, my name is " + this.name;
+  }
```

Comment:
I see that you added the speak property directly into the object - you can also assign a new property by doing

```
kennan.speak = function() {
  return "Hello, my name is Kennan!";
}
```
outside the initialized object.

The same goes for Antonietta exercise.

Overall, great job with the code! 👍